### PR TITLE
New Cheat Sheet: Spanish language

### PR DESCRIPTION
**Pull request by mayank7**
New Cheat Sheet: Spanish language

IA Page: https://duck.co/ia/view/spanish_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=spanish cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/spanish_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=spanish cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/spanish_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/language/spanish.json [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+         "key": "How are you?",
+         "val": "¿Cómo Estás?"
+      }, {
+         "key": "I'm fine, thanks!",
+         "val": "Estoy Bíen ¡Gracias!"
+      }, {
+         "key": "And you?",
+         "val": "¿Y Tú?"
+      }, {
+         "key": "Good/ So-So.",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/spanish.json]: @mayank7 Lets remove the period at end of key field.
Check other blocks too for the same.

Comment (mayank7) [inline share/goodie/cheat_sheets/json/language/spanish.json]: @jophab Why should I remove it ??
It is adding feeling/expression to the words or phrases. I think it is fine.

**Comment by GuiltyDolphin** [comment]
@mayank7 I'm going to close this after discussion on #2645 - feel free to go and give tips/suggestions over there!

**Comment by mayank7** [comment]
@GuiltyDolphin One question, the HackerEarth leaderboard was updated yesterday but it is not showing my name even if I have 85 PRs merged (zeroclickinfo-goodies). They are displaying people from 32 to 1 PRs, why my name is not there ??
https://www.hackerearth.com/sprints/open-source-india-hacks-2016/leaderboard

**Comment by GuiltyDolphin** [comment]
@mayank7 No idea - I don't really know that much about the HackerEarth competition - perhaps you should contact them?

**Comment by mayank7** [comment]
@GuiltyDolphin I already contacted them but haven't got any response corresponding for this.

**Pull request closed without merging**